### DISCUSSION TITLE
p/%E5%B7%A5%E4%BD%9C%E9%9A%8F%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [baixiaozhou/hexo-blog#4] [Announcements]
# p/%E5%B7%A5%E4%BD%9C%E9%9A%8F%E8%AE%B0/

记录工作中的一些日常操作

https://baixiaozhou.github.io/p/%E5%B7%A5%E4%BD%9C%E9%9A%8F%E8%AE%B0/

**Comment by panyy97** [comment]
左下角的黑白背景按钮点了没反应，速修
